<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Emprestimos</class>
 <widget class="QDialog" name="Emprestimos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EMPRÉSTIMO</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string>PESQUISA DO LIVRO</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>NOME DO LIVRO </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="NOME_LIVRO">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>11</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>NOME DO AUTOR</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="NOME_AUTOR">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>11</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>TRILHA                  </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="TRILHA">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>11</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>PESQUISAR</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string>PESQUISA DO LEITOR </string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NOME DO LEITOR </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="NOME_REQUERENTE">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>PESQUISAR</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string>EMPRÉSTIMO</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>DIAS PARA ENTREGA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber">
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DATA DA ENTREGA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCalendarWidget" name="CALENDAR"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ÍNDICE DO LIVRO</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="INDICE_LIVRO">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>11</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ÍNDICE DO LEITOR </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="INDICE_REQUERENTE">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>11</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CADASTRAR EMPRÉSTIMO</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
